package.json

```diff
@@ -60,7 +60,7 @@
     "npm": "^2.14.0"
   },
   "scripts": {
-    "build": "gulp",
+    "build": "git clone -b fix\/tracker-ebola git://github.com/frictionlessdata/jsontableschema-js.git node_modules/jsontableschema && cd node_modules/jsontableschema && npm install && npm run build && gulp",
```

Comment:
We need some sync with Python lib  (I'm just returning to this stuff today) but we under v1 so could do breaking changes on 0.x.0. I'm +1 to establish release process as soon as possible. Because for now I'm even not fully understand what we have on npm already.